link.json

```diff
@@ -0,0 +1,194 @@
+{
+    "appid": "c5a92c45-8c3e-417b-be46-01e8d7c677a6",
+    "name": "Link",
+    "description": "Generates a widget with clickable links",
+    "publisher": "Mhike",
+    "type": "devops",
+    "main_module": "link_connector.py",
+    "app_version": "1.0.0",
+    "utctime_updated": "2022-06-12T14:23:35.606945Z",
+    "package_name": "phantom_link",
+    "product_vendor": "Mhike",
+    "product_name": "Link",
+    "product_version_regex": ".*",
+    "min_phantom_version": "4.9.0",
+    "python_version": "3",
+    "logo": "logo_link.svg",
+    "logo_dark": "logo_link_dark.svg",
+    "license": "Copyright (c) Mhike, 2022",
+    "contributors": [
+        {
+            "name": "Mhike"
+        }
+    ],
+    "configuration": {
+        "https_port": {
+            "description": "Splunk SOAR HTTPS port if your instance uses one other than 443",
+            "data_type": "string",
+            "order": 0,
+            "name": "https_port",
+            "id": 0
+        },
+        "debug": {
+            "description": "Print debugging statements to log",
+            "data_type": "boolean",
+            "default": false,
+            "order": 1,
+            "name": "debug",
+            "id": 1
+        }
+    },
+    "actions": [
+        {
+            "action": "add url",
+            "description": "Generate a widget with clickable links",
+            "verbose": "Adds the specified links with the descriptor as the link text. Links are added to the existing link widget if one exists, otherwise a new widget will be generated.",
+            "type": "generic",
+            "identifier": "add_link",
+            "read_only": false,
+            "parameters": {
+                "url": {
+                    "data_type": "string",
+                    "order": 0,
+                    "description": "URL to be added, requires the description field",
+                    "name": "url",
+                    "id": 1,
+                    "param_name": "url"
+                },
+                "description": {
+                    "data_type": "string",
+                    "order": 1,
+                    "description": "Description to be added, requires the url field",
```

Comment:
might want to specify where its adding a description to:  "to be added to **X**"

Reply:
Thats a reasonable ask. To make this more explicit, it could be changed to "Description to be added as link text, requires the url field"